content/correspondence/reference/API endpoints/initialize/_index.en.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Initialize
+linktitle: Initialize
+description: Endpoint for initializing a correspondence.
+
+weight: 60
+toc: true
+---
+
+## Endpoint
+
+POST /correspondence/api/v1/correspondence
+
+## Description
+This endpoint prepares and queues a correspondence for sending. Before using this endpoint any correlating attachments must be uploaded beforehand to populate the "ExistingAttachments" field. This can be done by using the following endpoint:
+
+
+/correspondence/api/v1/attachment/{{attachmentId}}/upload     
```

Comment:
Still feels a bit wierd to say you must call a specific endpoint. Think a link to that enpoint would be better.